### ISSUE TITLE
'Download Metadata' button next to Download model

### ISSUE THREAD

**Issue by kalomaze** [oobabooga/text-generation-webui#3700]
**Description**

<img width="406" alt="image" src="https://github.com/oobabooga/text-generation-webui/assets/66376113/d9eb2e4c-ff65-453c-b74d-f1abdfe8f969">

Within the Model tab, you can download a model from scratch. However, it would be nice to have a feature where there is another option underneath labeled 'Download Metadata'. This would only grab the non-LFS files from the huggingface repository (aka, config.json and other relevant metadata.)

**Comment by Sanyam-jain30** [comment]
It looks interesting. Can I work on it?

**Comment by kalomaze** [comment]
Sure!

**Comment by jllllll** [comment]
Worth noting that the model downloader more or less does this when ran with the flag `--text-only`.
There just isn't a mechanism in the UI to use that flag.

**Comment by Sanyam-jain30** [comment]
I am pretty sure you know better than me. But, as fas as I know they are three conditions checked for the file:
1. Whether it is LFS file. 
2. Text only.
3. Not text only. 
So, I think just text only might not work. Maybe we can have a parameter for the present function that will not take LFS files. Moreover, I had just made a new model by replicating the previous model to avoid confusion and made changes wherever it was necessary.

**Comment by jllllll** [comment]
This is the only check in the the script that uses the `--text-only` flag:
https://github.com/oobabooga/text-generation-webui/blob/0576691538da4f93d50b9ee28c252eb91c4f0da3/download-model.py#L98-L114

While this check does not explicitly apply to LFS files, it does apply to all of the files that would be stored as LFS.
This has the effect of downloading all non-LFS files. The only downside to this is that it will still download LFS files of unknown file types. Whether or not having a flag for explicitly excluding LFS files is useful, I don't know. Doesn't hurt to add it as it is a fairly simple addition, given that their is already a check for LFS.

**Comment by Sanyam-jain30** [comment]
> https://github.com/oobabooga/text-generation-webui/blob/0576691538da4f93d50b9ee28c252eb91c4f0da3/download-model.py#L90-L91

Yes, for non-LFS to be more sure. I have just skipped this if condition. That was the only change required.
